deploy/deploy.gradle

```diff
@@ -99,8 +99,13 @@ afterEvaluate { project ->
                         dependencyNode.appendNode("scope", scope)
 
                     }
-                    configurations.api.getDependencies().each { dep -> addDependency(dep, "compile") }
-                    configurations.implementation.getDependencies().each { dep -> addDependency(dep, "runtime") }
+
+                    if (configurations.hasProperty("api")) {
```

Comment:
curious what would happen without the check?

Reply:
```
> Task :stripecardscan-tflite:generatePomFileForReleasePublication FAILED

FAILURE: Build failed with an exception.

* Where:
Script '/Users/brnunes/stripe/stripe-android/deploy/deploy.gradle' line: 102

* What went wrong:
Execution failed for task ':stripecardscan-tflite:generatePomFileForReleasePublication'.
> Could not apply withXml() to generated POM
   > Could not get unknown property 'api' for configuration container of type org.gradle.api.internal.artifacts.configurations.DefaultConfigurationContainer.

* Try:
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Exception is:
org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':stripecardscan-tflite:generatePomFileForReleasePublication'.
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.lambda$executeIfValid$1(ExecuteActionsTaskExecuter.java:147)
```